crates/bevy_asset/src/asset_server.rs

```diff
@@ -418,6 +444,7 @@ impl AssetServer {
         Ok(handles)
     }
 
+    /// Frees unused assets from their asset storages.
```

Comment:
```suggestion
    /// Frees unused assets, unloading them from memory
```
Better captures how / why this would be used, rather than focusing on an implementation detail.